src/Card/__stories__/Card.stories.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, boolean } from '@storybook/addon-knobs';
+
+import Card from '..';
+import CardReadme from '../README.md';
+
+storiesOf('Card', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      // Show readme before story
+      content: CardReadme,
+    },
+  })
+  .add('with properties', () => {
+    const isHeader = boolean('With Header', true, 'Layout');
+    const isFooter = boolean('With Footer', true, 'Layout');
+    return (
+      <div>
+        <Card height="350px" width="350px">
+          {isHeader && (
+            <Card.Header>
+              <h3 style={{ padding: '1.6rem 2rem' }}>Header</h3>
```

Comment:
I think the style could be directly in the Card.Header, see my comment on the component